app/src/main/java/org/kiwix/kiwixmobile/nav/destination/library/LocalLibraryFragment.kt

```diff
@@ -334,6 +336,23 @@ class LocalLibraryFragment : BaseFragment() {
       }, Throwable::printStackTrace
     )
 
+  private fun fileSelectActions() = zimManageViewModel.fileSelectActions
+    .observeOn(AndroidSchedulers.mainThread())
+    .filter { it === FileSelectActions.RequestDeleteMultiSelection }
```

Comment:
* Why we are using `===`? 
* Please directly import `RequestDeleteMultiSelection` to maintain the code flow like we are importing for other actions as shown in the below code snippet.

```
private val bookDelegate: BookOnDiskDelegate.BookDelegate by lazy {
    BookOnDiskDelegate.BookDelegate(
      sharedPreferenceUtil,
      { offerAction(RequestNavigateTo(it)) },
      { offerAction(RequestMultiSelection(it)) },
      { offerAction(RequestSelect(it)) }
    )
  }
  ```

Reply:
> * Why we are using `===`?

@MohitMaliFtechiz I thought it would be efficient to use referential equality for comparison than using '==' since RequestDeleteMultiSelection is an object instance.
Is it correct?

Reply:
Okay, Please fix the other part of the request changes.